docs/contribute.md

```diff
@@ -109,16 +82,11 @@ For every feature change or addition to `plone.api`, you must add documentation
 {doc}`plone:contributing/documentation/index`
 ```
 
-After adding or modifying documentation, you can build the documentation with the following command.
-
-```shell
-tox -e plone6docs
-```
-
-Alternatively, you can automatically reload changes to the documentation as you edit it in a web browser.
+When adding or modifying documentation, you can build the documentation with the following command.
+As you edit the documentation, the started process automatically reloads your changes in the web browser.
 
 ```shell
-tox -e livehtml
+make livehtml
 ```
 
 You can run a link checker on documentation.
```

Comment:
I added linkcheck for consistency, and pushed 8cc4b1d89db47d5b2b27b832464a8063074b6a71